_data/default_variants.yml

```diff
@@ -11,6 +11,7 @@ utilities:
   metabase: metabase
   sqlfluff: sqlfluff
   superset: apache
+  superset_ext: apache  
```

Comment:
I'm not sure if we want dupe `superset` and `superset_ext` entries in the long run here.

Reply:
@aaronsteers I synced with Pat on this. I think we're going to make a `meltano` variant for the extension variant and then when the time comes we can deprecate it and change the default apache variant to be EDK based.